engine/runtime-instrument-common/src/main/java/org/enso/interpreter/instrument/command/SetExecutionEnvironmentCommand.java

```diff
@@ -42,12 +42,12 @@ public Future<BoxedUnit> executeAsynchronously(RuntimeContext ctx, ExecutionCont
   private void setExecutionEnvironment(
       Runtime$Api$ExecutionEnvironment executionEnvironment, UUID contextId, RuntimeContext ctx) {
     var logger = ctx.executionService().getLogger();
+    ctx.jobControlPlane().abortJobs(contextId);
```

Comment:
This PR is moving `abortJobs` call outside of critical section, right? But the PR description says:

> We don't seem to run abortJobs under a lock

How's that accurate? We **do seem** to run `abortJobs` under a lock and this PR is changing that, am I not right?

Reply:
I meant in other locations that call `abortJobs`